src/D2L.Bmx/OktaAuthenticator.cs

```diff
@@ -205,22 +205,9 @@ async Task GetSessionCookieAsync() {
 				}
 			}
 		} catch( TaskCanceledException ) {
-			consoleWriter.WriteWarning( $"""
-				Timed out when trying to automatically sign in to Okta. Check if the org '{orgUrl}' is correct.
-				If you have to run BMX with elevated privileges, and aren't concerned with the security of {orgUrl.Host},
-				consider running the command again with the '--experimental-bypass-browser-security' flag.
-				"""
-			);
-		} catch( TargetClosedException ) {
-			consoleWriter.WriteWarning( """
-				Failed to automatically sign in to Okta as BMX is likely being run with elevated privileges.
-				If you have to run BMX with elevated privileges, and aren't concerned with the security of {orgUrl.Host},
-				consider running the command again with the '--experimental-bypass-browser-security' flag.
```

Comment:
since https://github.com/Brightspace/bmx/pull/482, we check for elevated permission beforehand and short circuit the logic, so messages here are unnecessary